src/services/videoconference/index.js

```diff
@@ -507,13 +497,121 @@ class CreateVideoconferenceService {
 	}
 }
 
+const jwtVerify = promisify(jwt.verify);
+
+class RecordingReadyVideoconferenceService {
+	constructor(app) {
+		this.app = app;
+		this.docs = {};
+	}
+
+	async create(data, params) {
+		const { route } = params;
+		const { meeting_id: meetingId, record_id: recordId } = await jwtVerify(data.signed_parameters, SALT);
+
+		const { response } = await server.recording.getRecordings({ recordId });
+
+		const entries = response.recordings[0].recording;
+		const isPresentation = (pb) => pb.format[0].type[0] === 'presentation';
+		const recording = entries.find(({ playback: [pb] }) => isPresentation(pb));
+		const playback = recording.playback.find(isPresentation).format[0];
+
+		// Update videoconference, store recordId
+		const conference = await VideoconferenceModel.findByIdAndUpdate(route.id, { recordId }).orFail().exec();
+
+		// Connect to the message broker
+		const channel = await rabbitmq.createChannel();
+
+		// Ensure the queue exists
+		await channel.assertQueue(RECORDER_QUEUE, { durable: true });
+
+		// Send message to the queue for exporting the playback to a video
+		const payload = {
+			url: playback.url[0],
+			duration: Math.ceil((recording.endTime[0] - recording.startTime[0]) / 1000),
+			vid: conference.id,
+		};
+		const buffer = Buffer.from(JSON.stringify(payload));
+		await channel.sendToQueue(RECORDER_QUEUE, buffer, { persistent: true });
+
+		await channel.close();
+
+		return { ok: true };
```

Comment:
BBB doesn't really care about the response body. It just expects an "ok" (`2xx`) status code.